assets/src/styles/blocks/Articles.scss

```diff
@@ -119,10 +121,10 @@
 .article-list-item-content {
   --block-articles--article--content-- {
     font-size: inherit;
+    width: 80%;
   }
   display: none;
   margin: $space-xs 0 0;
```

Comment:
Can't this margin also be a variable? 🤔

Reply:
It can, but for now I only added variables for spacing changes where a theme needs these (and it made sense). We probably want to clean up some blocks CSS after we converted legacy themes, so the spacing CSS might look different.